### ISSUE TITLE
Failed to detect key of advanced mod descriptions

### ISSUE THREAD

**Issue by tfp** [SnosMe/awakened-poe-trade#447]
And I can't set it manually. It's stuck on `ALT`.
Path to the config file is set. The key I'm using is keyboard layout dependant. It's `<` but it's shown as `\` ingame.
According to the config file it's `highlight=226`

**Comment by thisVioletHydra** [comment]
> And I can't set it manually. It's stuck on `ALT`.
> Path to the config file is set. The key I'm using is keyboard layout dependant. It's `<` but it's shown as `\` ingame.
> According to the config file it's `highlight=226`

i set **H**, and now work =\

**Comment by tfp** [comment]
Yes, I could change the key, but there is no real other good key for my usecase. And it indicates that there is problem with handling virtual/oem-spefic keys in the code.

**Comment by tfp** [comment]
Digged a bit into the code. You try to  read the stored keyboard code and map that to an actual character, as robot.js wants to have a character instead of a code. But your mapping list does not include OEMs as these could be keyboard layout dependant. I think the simplest solution would be to allow to overwrite the key in the settings.

**Comment by thisVioletHydra** [comment]
> Digged a bit into the code. You try to read the stored keyboard code and map that to an actual character, as robot.js wants to have a character instead of a code. But your mapping list does not include OEMs as these could be keyboard layout dependant. I think the simplest solution would be to allow to overwrite the key in the settings.

https://github.com/SnosMe/awakened-poe-trade/issues/438#issuecomment-891351749
i think some keycodes not supported 🥴

**Comment by keysen** [comment]
Hey, same issue for me with « Mouse 5 » used to highlight mods on items.

**Comment by SnosMe** [comment]
At the moment, I don't know how to pair robotjs, poe config, electron global hotkeys, web input events in settings. Open for PR

**Comment by NGTagger** [comment]
Was looking at this, as making hotkeys was real confusing for me (Nordic keyboard layout - specifically Danish).
I've used **Shift+Tilde** key (to the left of 1 on my layout) for a long time, when wanting to open the "main overlay", yet that doesn't get picked up as the "correct key" in the app. 

If I want to use that key; I need to press the keys **Shift+'** (which the app takes as **Shift+Backslash** - whereas **Shift+Tilde** gets picked up as **Shift+Backquote**), which is right in-between the Ø and Enter key on my layout.
..yet it actually opens when I press **Shift+Tilde**, despite using another key for setting the keybind. 

It's a bit confusing, the way it's apparently set up.

**Comment by SuperJelle** [comment]
Key mapping is still broken for Nordic keyboard layouts. I have to press ' to map the ½ key on my keyboard.
